frontend/leaderboard/src/Sheets.re

```diff
@@ -0,0 +1,87 @@
+open Bindings;
+
+let clientId = "";
+let clientSecret = "";
+let redirectURI = "";
```

Comment:
These will probably end up being a json file that gets loaded, not hardcoded in the code